src/components/MoviesNavbarSortBy/index.tsx

```diff
@@ -2,17 +2,33 @@ import React from 'react';
 import cn from 'classnames';
 
 import Dropdown from 'components/_common/Dropdown';
+import useDropdown from 'hooks/useDropdown';
 import styles from './styles.module.scss';
 
+const options = ['release date', 'rating', 'best 100'];
+
```

Comment:
I would keep such kind of options in some config.  But it's up to you and not necessary for now

Reply:
Yep, good point, I will move in the next task.